README.md

```diff
@@ -85,12 +85,13 @@ GuardDog comes with 2 types of heuristics:
 ### Running a local version of GuardDog
 
 * Clone the repository
-* Create a virtualenv: `python3 -m venv venv && source venv/bin/activate`
+* Install needed virtualenv: ` $ poetry install `
```

Comment:
Is there any way we could avoid enforcing usage of poetry? Many people (especially outside of development) don't have it installed

Reply:
I think it's possible- maybe all it will take it specifying a flag for venv to use a higher version of python (I was getting an error on using match as a keyword in the project code in venv). I'll try that locally. 

Even so, there is redundancy in the dependency management files, since the repo has [poetry.lock](https://github.com/DataDog/guarddog/blob/main/poetry.lock) and [requirements.txt](https://github.com/DataDog/guarddog/blob/main/requirements.txt). Would it be better to remove the use of poetry all together? I do think the README local version instructions should also specify best ways to install dependencies too.